Formula/p/python-no-gil.rb

```diff
@@ -0,0 +1,518 @@
+class PythonNoGil < Formula
```

Comment:
```suggestion
class PythonFreethreading < Formula
```
This seems to be the more commonly used name.